android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/BackButton.kt

```diff
@@ -23,22 +21,15 @@ fun BackButton(
     modifier: Modifier,
     onClick: () -> Unit
 ) {
-    Button(
-        onClick = onClick,
+    Image(
+        painter = painterResource(id = R.drawable.ic_arrow_right),
+        contentDescription = null,
+        colorFilter = ColorFilter.tint(White),
+        contentScale = ContentScale.FillWidth,
         modifier = modifier
             .wrapContentSize()
-            .rotate(Rotate180), // switch Right Arrow to Back button
-        contentPadding = PaddingValues(dimensions.paddingMedium),
-        colors = ButtonDefaults.buttonColors(
-            backgroundColor = Transparent
-        ),
-        elevation = null
-    ) {
-        Image(
-            painter = painterResource(id = R.drawable.ic_arrow_right),
-            contentDescription = null,
-            colorFilter = ColorFilter.tint(Color.White),
-            contentScale = ContentScale.FillWidth
-        )
-    }
+            .rotate(Rotate180) // switch Right Arrow to Back button
```

Comment:
I don't think we need this comment here 👍

Reply:
@blyscuit Hmm, it's still a good signal for others to understand why I need to make a rotation 😛